### PR TITLE
fix(business): update SbbTableDataSource data on disconnected state

### PR DESCRIPTION
**Pull request by kyubisation**

### CODE REVIEW

File: src/angular-business/table/table/table-data-source.ts [hunk 1]
```diff
@@ -372,6 +391,10 @@ export class SbbTableDataSource<
    * @docs-private
    */
   connect() {
+    if (!this._renderChangesSubscription) {
```
Comment (jeripeierSBB) [inline src/angular-business/table/table/table-data-source.ts:394]: Don't we have to support disconnect?

  /**
   * Used by the SbbTable. Called when it connects to the data source.
   * @docs-private
   */
  connect() {
    if (!this._renderChangesSubscription) {
      this._updateChangeSubscription();
    }

    return this._renderData;
  }

File: src/angular-business/table/table/table-data-source.ts [hunk 2]
```diff
@@ -372,14 +391,21 @@ export class SbbTableDataSource<
...
+      this._updateChangeSubscription();
+    }
+
     return this._renderData;
   }
 
   /**
    * Used by the Table. Called when it is destroyed. No-op.
```
Comment (jeripeierSBB) [inline src/angular-business/table/table/table-data-source.ts:402]:
```suggestion
   * Used by the SbbTable. Called when it disconnects from the data source.
```

File: src/angular-business/table/table/table-data-source.ts (hunk 1 above)
Comment (kyubisation) [reply src/angular-business/table/table/table-data-source.ts:394]: You're absolutey right

**Review by jeripeierSBB — APPROVED** [review]
LGTM

But I did no test :)

**Pull request merged**